Tests/MongoSwiftTests/TestUtils.swift

```diff
@@ -113,6 +113,22 @@ extension MongoClient {
 
         return true
     }
+
+    internal convenience init(options: ClientOptions? = nil) throws {
+        try self.init(connectionString: getConnStr(), options: options)
+    }
+
+    internal convenience init() throws {
+        try self.init(connectionString: getConnStr())
+    }
+}
+
+func getConnStr() -> String {
+    if let connStr = ProcessInfo.processInfo.environment["MDB_CONN_STR"] {
+        return connStr
+    } else {
+        return "mongodb://localhost:27017"
```

Comment:
libmongoc defaults to [`mongodb://127.0.0.1/`](https://github.com/mongodb/mongo-c-driver/blob/master/src/libmongoc/src/mongoc/mongoc-client.c#L890) in its client constructor. I realize that Swift uses [`mongodb://localhost:27017`](https://github.com/mongodb/mongo-swift-driver/blob/v0.0.5/Sources/MongoSwift/MongoClient.swift#L124), but I would suggest sticking to libmongoc's default. Perhaps that belongs in a new ticket.

Despite the client constructor default, libmongoc does say its test suite defaults to `localhost` in [`CONTRIBUTING.md`](https://github.com/mongodb/mongo-c-driver/blob/master/CONTRIBUTING.md#testing) if the `MONGOC_TEST_HOST` variable is not defined. PHP's test suite always defaults to `mongodb://127.0.0.1/` (see: [here](https://github.com/mongodb/mongo-php-driver/blob/1.5.3/tests/utils/basic.inc#L5)).

I'm a proponent of using the IPv4 address over localhost as some platforms will resolve `localhost` as an IPv6 address, and some older versions of MongoDB may not listen on IPv6 by default. For that reason, the IPv4 address is more portable IMO. In both cases, the port isn't necessary in the default URI as libmongoc already supplies `27017` as its default port if one isn't specified in the URI.

Reply:
TIL! Done.